apps/blog/src/app/post.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+import { AngularFireDatabase, AngularFireList } from '@angular/fire/database';
+import { Observable } from 'rxjs';
+import { Post } from './form/form.component';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class PostService {
+    repo$: AngularFireList<Post> = this.database.list('/posts', (ref) => {
+        return ref.orderByChild('hidden').equalTo(null);
+      });
+
+    constructor(
+        private database: AngularFireDatabase) {
+    }
+
+    removePost(id: string) {
+        return this.database.object(`posts/${id}`).remove();
```

Comment:
can you abstract out 
 this.database.object(`posts/${id}`)?
As it is used 3 times